src/compas_view2/shaders/120/mesh.vert

```diff
@@ -5,6 +5,7 @@ attribute vec3 color;
 
 uniform mat4 projection;
 uniform mat4 viewworld;
+uniform mat4 transform;
```

Comment:
is transform not a reserved keyword in some flavours of glsl?

Reply:
Says that all glsl reserved names start with `gl_`

https://www.khronos.org/opengl/wiki/Core_Language_(GLSL)#:~:text=to%20be%20compatibility%20.-,Reserved%20names,as%20identifiers%20in%20any%20context.

Reply:
ok